packages/jellyfish-api-core/src/category/blockchain.ts

```diff
@@ -43,3 +76,73 @@ export interface BlockchainInfo {
   }
   warnings: string
 }
+
+export interface BlockBase {
+  hash: string
+  confirmations: number
+  strippedsize: number
+  size: number
+  weight: number
+  height: number
+  masternode: string
+  minter: string
+  mintedBlocks: any
+  stakeModifier: any
+  version: number
+  versionHex: string
+  merkleroot: string
+  time: number
+  mediantime: number
+  bits: string
+  difficulty: number
+  chainwork: string
+  nTx: number
+  previousblockhash: string
+  nextblockhash: string
+}
+
+export interface BlockVerbo1 extends BlockBase {
+  tx: string[]
+}
+
+export interface BlockVerbo2 extends BlockBase {
+  tx: RawTx[]
+}
```

Comment:
you might want to use generics also to keep it simple.

`Block<RawTx>`
`Block<string>`

```ts
export interface Block<T> {
  //...
  chainwork: string
  tx: T[]
  nTx: number
  //...
}
```